yarn.lock

```diff
@@ -13179,6 +13184,16 @@ react-resizable@^3.0.4:
     prop-types "15.x"
     react-draggable "^4.0.3"
 
+react-resize-panel@^0.3.5:
+  version "0.3.5"
+  resolved "https://registry.yarnpkg.com/react-resize-panel/-/react-resize-panel-0.3.5.tgz#43aa3450bf5b5a2566b40c4201445ced96c2a905"
+  integrity sha512-iyHOFTrSt+WV4Ilzi81x6KH3FU7VsGP736rmxepwGrgAEATmCvXzZdluTm3NpsptP7aC3hLODmXwnxusyA393A==
+  dependencies:
+    cash-dom "^4.1.5"
+    classnames "^2.2.6"
+    lodash.debounce "^4.0.8"
+    react-draggable "^4.0.3"
```

Comment:
> `react-draggable "^4.0.3"`

This is doing the heavy lifting. It's already in our yarn.lock file because of `react-grid-layout`.


`react-grid-layout` is better suited for dashboards afaik. I couldn't get it to do fluid column sizes, it likes to snap to a grid.